main.go

```diff
@@ -25,7 +28,19 @@ func main() {
 
 	scanner := bufio.NewScanner(os.Stdin)
 	for scanner.Scan() {
-		if err := verifyToken(scanner.Text()); err != nil {
+		stdinArgs := strings.Split(scanner.Text(), "||")
+		var checkPath = false
+		if len(stdinArgs) == 1 {
+			checkPath = false
+		} else if len(stdinArgs) == 2 {
+			checkPath = true
+			requestPath = stdinArgs[1]
+		} else {
+			fmt.Fprintln(os.Stdout, "INVALID NUMBER OF STDIN ARGS SUPPLIED")
```

Comment:
If it's an error, should we be using fmt.Errorf?

Reply:
Infact, might need to think about this. If I remember correctly, some error paths need to return errors via STDOUT if I remember correctly? What happens when we return a non-zero exit code when this is executed in the contact of an Apache rewrite rule?

Reply:
I believe we did initially use this but due to how we want apache to handle the output from this we want it to just be a standard output